health/probe/plugin_impl_probes.go

```diff
@@ -75,11 +75,7 @@ func (p *Plugin) Init() (err error) {
 				HTTPport: httpPort,
 			},
 		}
-		err := p.HTTP.Init()
-		if err != nil {
-			return err
-		}
-		err = p.HTTP.AfterInit()
+		err := p.HTTP.ExternalInit()
```

Comment:
No need to do that - just rebase. Yesterday we have cleaned up this - basically removed nested plugin initialization.